about.html

```diff
@@ -0,0 +1,173 @@
+        <nav id="menu">
+          <span id="menu-close-btn"><i class="fa-solid fa-xmark"></i></span>
+          <a href="./index.html" class="home-link">Home</a>
+          <a href="./index.html#program">Program</a>
+          <a href="./index.html#speakers">Speakers</a>
+          <a href="./index.html#sponsors">Sponsors</a>
+          <a href="#" class="active">About</a>
+          <a href="#">
+            <span class="cc-campaign">Awards</span>
+          </a>
+        </nav>
+        <a href="#" id="menu-toggler">
+          <i class="fa-solid fa-bars"></i>
+        </a>
+      </div>
+    </header>
+
+    <main>
+      <!-- Hero - section -->
+      <div class="container">
+        <div id="hero-about" class="wrapper">
+          <img src="images/hero-bg.png" alt="hero-bg" class="hero-bg" />
+          <h1>
+            ETHNIC AND <br />
+            RELIGIOUS <br />CONFLICT <br />
+            RESOLUTION AND <br >PEACEBUILDING
+          </h1>
+          <h2>2022 International Conference on</h2>
+
+          <div class="description">
+            <p>To increase our understanding of ethnic, 
+                racial and religious conflicts in countries around the world, 
+                the conference will consider submissions from multidisciplinary fields of study and practice. 
+                Qualitative, quantitative or mixed methods research studies from university scholars and researchers are accepted. 
+                Case studies, lessons learned, success stories, policy analysis or design, and best practices from policy makers,
+                 practitioners, religious leaders and indigenous peoples are also accepted
+
+
+            </p>
+          
+          </div>
+          <p>
+            For queries please contact at <br />
+            <a href=" info@panafricanyouthconference.org">
+                info@panafricanyouthconference.org</a
+            >
+          </p>
+        </div>
+      </div>
+
+      <section id="standOut">
+        <div class="wrapper">
+          <h2 class="section-heading">
+            Pan-African youth conference Logo
+          </h2>
+          <p>
+            The logo of Pan-African Youth conference was decided throu the logo competition from 8,june to 7 july.<br >
+            after online public voting and selection commitee's judging process, the logo from Nahom minas won the prize.<br />
+          </p>
+          <img loading="lazy"  src="https://www.panafricanyouthconference.org/wp-content/uploads/2022/01/PAYC-Logo-v2.1.png" alt="" title="PAYC-Logo v2.1" srcset="https://www.panafricanyouthconference.org/wp-content/uploads/2022/01/PAYC-Logo-v2.1.png 341w, https://www.panafricanyouthconference.org/wp-content/uploads/2022/01/PAYC-Logo-v2.1-300x145.png 300w" sizes="(max-width: 341px) 100vw, 341px" class="wp-image-243">
```

Comment:
Kindly, add some `padding-top` and `padding-bottom` for the `<p>` tag to give some space. Also, add a `border` for the image and the after a `horizontal` line as shown here in the design provided:

<img width="854" alt="image" src="https://user-images.githubusercontent.com/34578631/170122126-224e13b0-8da9-462c-81ae-da0312db2af7.png">